public/app/features/alerting/unified/components/rule-editor/FolderAndGroup.tsx

```diff
@@ -403,37 +408,35 @@ function EvaluationGroupCreationModal({
               data-testid={selectors.components.AlertRules.newEvaluationGroupName}
               className={styles.formInput}
               autoFocus={true}
-              id={'group'}
+              id={evaluationGroupNameId}
               placeholder="Enter a name"
               {...register('group', { required: { value: true, message: 'Required.' } })}
             />
           </Field>
 
           <Field
             error={formState.errors.evaluateEvery?.message}
-            invalid={Boolean(formState.errors.evaluateEvery) ? true : undefined}
             label={
               <Label htmlFor={evaluateEveryId} description="How often all rules in the group are evaluated.">
                 Evaluation interval
               </Label>
             }
+            invalid={Boolean(formState.errors.evaluateEvery)}
           >
-            <Stack direction="column">
-              <Input
-                data-testid={selectors.components.AlertRules.newEvaluationGroupInterval}
-                className={styles.formInput}
-                id={evaluateEveryId}
-                placeholder={DEFAULT_GROUP_EVALUATION_INTERVAL}
-                {...register(
-                  'evaluateEvery',
-                  evaluateEveryValidationOptions<{ group: string; evaluateEvery: string }>(groupRules)
-                )}
-              />
-              <Stack direction="row" alignItems="flex-end">
-                <EvaluationGroupQuickPick currentInterval={evaluationInterval} onSelect={setEvaluationInterval} />
-              </Stack>
-            </Stack>
+            <Input
+              data-testid={selectors.components.AlertRules.newEvaluationGroupInterval}
+              className={styles.formInput}
+              id={evaluateEveryId}
+              placeholder={DEFAULT_GROUP_EVALUATION_INTERVAL}
+              {...register(
+                'evaluateEvery',
+                evaluateEveryValidationOptions<{ group: string; evaluateEvery: string }>(groupRules)
+              )}
+            />
           </Field>
+
+          <EvaluationGroupQuickPick currentInterval={evaluationInterval} onSelect={setEvaluationInterval} />
```

Comment:
Having a `Stack` inside the `Field` component results in incorrect props being spread onto the element. We were ending up with `invalid` going onto the `div` of the `Stack`, which caused a console.error, and failed tests when I was trying to work with this behaviour (as the act of clearing and typing into the interval input will temporarily mark it as invalid and show the error - which is fine, but the `invalid` state was causing the test to fail)